web/src/main/java/se/inera/intyg/rehabstod/web/controller/api/SickLeaveController.java

```diff
@@ -40,15 +43,17 @@ public class SickLeaveController {
     private GetActiveSickLeavesResponseService getActiveSickLeavesResponseService;
     @Autowired
     private GetSickLeaveSummaryService getSickLeaveSummaryService;
-
     @Autowired
     private PopulateFiltersService populateFiltersService;
+    @Autowired
+    private PdlLogSickLeavesService pdlLogSickLeavesService;
 
     private static final boolean INCLUDE_PARAMETERS = true;
 
     @RequestMapping(value = "/active", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE)
     public SickLeavesResponseDTO getSickLeavesForUnit(@RequestBody SickLeavesFilterRequestDTO request) {
         final var response = getActiveSickLeavesResponseService.get(request, INCLUDE_PARAMETERS);
+        pdlLogSickLeavesService.log(response.getContent(), ActivityType.READ, ResourceType.RESOURCE_TYPE_SJUKFALL);
```

Comment:
I don't think we should have this logic inside the controller. Should be part of the service layer.